src/components/elements/SnackbarAlert.tsx

```diff
@@ -0,0 +1,40 @@
+import { Alert, AlertProps, AlertTitle, Snackbar } from '@mui/material';
+
+import { ReactNode } from 'react';
+
+interface Props {
+  open: boolean;
+  onClose: VoidFunction;
+  children?: ReactNode;
+  alertProps?: AlertProps;
+  title?: string;
+}
+const SnackbarAlert: React.FC<Props> = ({
```

Comment:
Ideally we'd use this in the existing Apollo Alert but refactoring that felt like a bit too much scope creep for this branch